android/build.gradle

```diff
@@ -5,21 +5,27 @@ import org.gradle.util.VersionNumber
 // sub-projects/modules.
 
 buildscript {
+    ext {
+        kotlinVersion = '1.6.10'
+    }
     repositories {
         google()
         jcenter()
     }
     dependencies {
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
         classpath 'com.android.tools.build:gradle:4.1.2'
         classpath 'com.google.gms:google-services:4.3.4'
         classpath 'com.google.firebase:firebase-crashlytics-gradle:2.4.1'
+
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.6.10"
     }
 }
 
 ext {
     buildToolsVersion = "30.0.3"
     compileSdkVersion = 31
-    minSdkVersion    = 23
+    minSdkVersion    = 26
```

Comment:
We cannot do this, that would cut support for many old-but-still-used Android versions.

Reply:
![image](https://user-images.githubusercontent.com/26413496/148807040-92064e16-e112-4db5-b7a0-a6d73e119c10.png)

This was the reason behind this change. I cannot build the app unless version is 26.

Reply:
This is now reversed to 23.